var/spack/repos/builtin/packages/py-fenics-dolfinx/package.py

```diff
@@ -30,34 +33,42 @@ class PyFenicsDolfinx(PythonPackage):
     depends_on("python@3.7:3.10", when="@0.4", type=("build", "run"))
 
     depends_on("fenics-dolfinx@main", when="@main")
+    depends_on("fenics-dolfinx@0.7.2", when="@0.7.2")
+    depends_on("fenics-dolfinx@0.7.1", when="@0.7.1")
+    depends_on("fenics-dolfinx@0.7.0", when="@0.7.0")
     depends_on("fenics-dolfinx@0.6.0", when="@0.6.0")
     depends_on("fenics-dolfinx@0.5.1", when="@0.5.1")
     depends_on("fenics-dolfinx@0.5.0", when="@0.5.0")
     depends_on("fenics-dolfinx@0.4.1", when="@0.4.1")
 
     depends_on("fenics-basix@main", type=("build", "link"), when="@main")
+    depends_on("fenics-basix@0.7.0:0.7", type=("build", "link"), when="@0.7.0:0.7")
     depends_on("fenics-basix@0.6.0:0.6", type=("build", "link"), when="@0.6.0:0.6")
     depends_on("fenics-basix@0.5.1:0.5", type=("build", "link"), when="@0.5.0:0.5")
     depends_on("fenics-basix@0.4.2", type=("build", "link"), when="@0.4.1")
 
     depends_on("py-fenics-ffcx@main", type=("build", "run"), when="@main")
+    depends_on("py-fenics-ffcx@0.7", type=("build", "run"), when="@0.7")
     depends_on("py-fenics-ffcx@0.6", type=("build", "run"), when="@0.6")
     depends_on("py-fenics-ffcx@0.5.0.post0", type=("build", "run"), when="@0.5.0:0.5")
     depends_on("py-fenics-ffcx@0.4.2", type=("build", "run"), when="@0.4.1")
 
     depends_on("py-fenics-ufl@main", type=("build", "run"), when="@main")
+    depends_on("py-fenics-ufl@2023.2.0", type=("build", "run"), when="@0.7")
     depends_on("py-fenics-ufl@2023.1", type=("build", "run"), when="@0.6")
     depends_on("py-fenics-ufl@2022.2.0", type=("build", "run"), when="@0.5.0:0.5")
     depends_on("py-fenics-ufl@2022.1.0", type=("build", "run"), when="@0.4.1")
 
+    # See python/pyproject.toml
```

Comment:
This file only exists in main, not any of the older releases

Reply:
I didn't add these comments, would rather remove them.

Reply:
The line should remain (was requested by Spack reviewers of py-fenics-basix PRs). I think the query is on how it's popped up in this PR.